src/controller/python/test/unit_tests/test_generated_clusterobjects.py

```diff
@@ -0,0 +1,100 @@
+import unittest
+import chip.clusters as Clusters
+from rich.pretty import pprint
+from chip.clusters.Types import Nullable, NullValue
+
+'''
+This file contains tests for validating the generated cluster objects by running encoding and decoding
+through them symmetrically to exercise both pathways.
+'''
+
+
+class TestGeneratedClusterObjects(unittest.TestCase):
+    def CheckData(self, expected):
```

Comment:
Would prefer a encode test + decode test, so the test will be stronger.

Consider the case that someone touched the encoding code and it does not match the TLV's null value repr, we should be able to reject such case.